src/azure-cli/azure/cli/command_modules/backup/_params.py

```diff
@@ -21,18 +21,19 @@
 # ARGUMENT DEFINITIONS
 
 allowed_container_types = ['AzureIaasVM']
-allowed_workload_types = ['VM', 'AzureFileShare', 'SAPHANA', 'MSSQL', 'SAPHanaDatabase', 'SQLDataBase']
-allowed_azure_workload_types = ['MSSQL', 'SAPHANA', 'SAPASE', 'SAPHanaDatabase', 'SQLDataBase']
+allowed_workload_types = ['VM', 'AzureFileShare', 'SAPHANA', 'SAPHanaDatabase', 'MSSQL', 'SQLDataBase']
+allowed_azure_workload_types = ['MSSQL', 'SQLDataBase', 'SAPHANA', 'SAPHanaDatabase', 'SAPASE']
 allowed_backup_management_types = ['AzureIaasVM', 'AzureStorage', 'AzureWorkload']
 allowed_extended_backup_management_types = allowed_backup_management_types + ['MAB']
-allowed_protectable_item_type = ['SQLAG', 'SQLInstance', 'SQLDatabase', 'HANAInstance', 'SAPHanaDatabase', 'SAPHanaSystem']
+allowed_protectable_item_type = ['SQLAG', 'SQLInstance', 'SQLDatabase', 'HANAInstance', 'SAPHanaSystem', 'SAPHanaDatabase']
```

Comment:
Why do we need to exchange the order of enumeration values?

Reply:
There is no need, it just happened while reverting an older commit. Should I change it back to what it was earlier, or is this fine?

Reply:
Please roll back to reduce the unnecessary modifications

Reply:
Done.